aldryn_faq/views.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import unicode_literals
-from django.core.urlresolvers import resolve
```

Comment:
This is actually unrelated to this PR's subject, but I'm just cleaning up an unused import here.